src/utils/notification.service.js

```diff
@@ -0,0 +1,12 @@
+import { notification } from 'antd'
+
+class NotificationService {
+  openNotification(type, message, description, duration) {
```

Comment:
maybe it will be more convenient to pass this 4 arguments as an object? I've already seen some code places, where you pass '' to second param. this isn't good idea